src/App.tsx

```diff
@@ -1,26 +1,229 @@
-/* eslint-disable max-len */
-/* eslint-disable jsx-a11y/control-has-associated-label */
-import React from 'react';
-import { UserWarning } from './UserWarning';
-
-const USER_ID = 0;
+/* eslint-disable jsx-a11y/label-has-associated-control */
+import React, { useEffect, useState, useRef } from 'react';
+import { USER_ID } from './api/todos';
+import { Todo } from './types/Todo';
+import * as postService from './api/todos';
+import classNames from 'classnames';
+import { Filter } from './types/Filter';
+import { Header } from './components/Header';
+import { TempToDo } from './components/TempToDo';
+import { Footer } from './components/Footer';
+import { ToDoItem } from './components/ToDoItem';
 
 export const App: React.FC = () => {
+  // #region loadToDOs
+  const [todos, setToDos] = useState<Todo[]>([]);
+  const [, setLoading] = useState<boolean>(false);
```

Comment:
During update or delete multiply todos loader should be on each todo which in processing on server.
Instead boolean state use state whick will contain array of todo ids, whick now in processing on server.